### PR TITLE
Support UNTESTED test state

### PR DESCRIPTION
**Pull request by odubaj**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,7 +50,11 @@ dependencies {
     } else {
         compile 'com.github.reportportal:commons:def053af'
         compile 'com.github.reportportal:commons-rules:331c402'
-        compile 'com.github.reportportal:commons-model:22ccaac'
+        //compile fileTree(dir: '/home/odubaj/DIPLOMKA/commons-rules/build/libs', include: ['*.jar'])
```
Comment (Yumfriez) [inline build.gradle:53]: remove comments please

File: build.gradle [hunk 2]
```diff
@@ -50,7 +50,11 @@ dependencies {
...
+        compile 'com.github.odubaj:commons-model:5b47278'
+        //compile fileTree(dir: '/home/odubaj/DIPLOMKA/commons-model/build/libs', include: ['*.jar'])
```
Comment (Yumfriez) [inline build.gradle:55]: remove comments please

File: build.gradle [hunk 3]
```diff
@@ -50,7 +50,11 @@ dependencies {
     } else {
         compile 'com.github.reportportal:commons:def053af'
         compile 'com.github.reportportal:commons-rules:331c402'
-        compile 'com.github.reportportal:commons-model:22ccaac'
+        //compile fileTree(dir: '/home/odubaj/DIPLOMKA/commons-rules/build/libs', include: ['*.jar'])
+        compile 'com.github.odubaj:commons-model:5b47278'
```
Comment (Yumfriez) [inline build.gradle:54]: will be replaced on our dependency after approval

File: build.gradle [hunk 4]
```diff
@@ -50,7 +50,11 @@ dependencies {
...
+        //compile fileTree(dir: '/home/odubaj/DIPLOMKA/commons-model/build/libs', include: ['*.jar'])
+        compile 'org.springframework.security:spring-security-core'
+        compile 'org.springframework:spring-webmvc'
```
Comment (Yumfriez) [inline build.gradle:57]: this dependencies are not required here

File: project-properties.gradle [hunk 5]
```diff
@@ -2,7 +2,7 @@ sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 
 def commonScriptsUrl = 'https://raw.githubusercontent.com/reportportal/gradle-scripts/'
-def migrationsScriptsUrl = 'https://raw.githubusercontent.com/reportportal/migrations/'
+def migrationsScriptsUrl = 'https://raw.githubusercontent.com/odubaj/migrations/'
```
Comment (Yumfriez) [inline project-properties.gradle:5]: will be replaced on our dependency after approval

**Review by Yumfriez — CHANGES_REQUESTED** [review]
Also update your branch with latest updates please

**Comment by odubaj** [comment]
Requested changes resolved, thanks!

**Pull request merged**